modules/navier_stokes/src/actions/NSFVAction.C

```diff
@@ -489,6 +489,8 @@ NSFVAction::NSFVAction(InputParameters parameters)
     _turbulence_handling(getParam<MooseEnum>("turbulence_handling")),
     _porous_medium_treatment(getParam<bool>("porous_medium_treatment")),
     _porosity_name(getParam<MooseFunctorName>("porosity")),
+    _smooth_porosity_functor_name(isParamValid("porosity_smoothing_layers") ? NS::smoothed_porosity
+                                                                            : _porosity_name),
```

Comment:
I think a name like `_flow_kernel_porosity_functor_name` might be better? Because this may or may not represent a smoothed porosity functor

Reply:
_flow_porosity_functor?

Reply:
i like that